lib/api/api.js

```diff
@@ -44,7 +44,7 @@ auth.setHandler(vault);
 const api = {
     callApiMethod(apiMethod, request, log, callback, locationConstraint) {
         // no need to check auth on website or cors preflight requests
-        if (apiMethod === 'websiteGet' || apiMethod === 'websiteHead' ||
+        if (apiMethod === 'websiteHead' ||
```

Comment:
unfortunately you might be able to expect most of the changes you made to also apply to websiteHead, and it could be a little strange to have different behavior between websiteHead/Get. are you planning to refactor wesiteHead in separate PR or commit?